Gemfile

```diff
@@ -27,3 +27,5 @@ end
 gem "rails", rails
 
 gem "an_engine", path: "./spec/an_engine"
+
+gem "addressable"
```

Comment:
Can you move this to https://github.com/openstax/path_prefixer/blob/master/path_prefixer.gemspec? Otherwise bundler will not bundle it correctly. Use `s.add_dependency`

Reply:
Done